drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -207,6 +208,10 @@ def incoming_referral_postings(**args)
       )
     end
 
+    def staff_assignments_enabled
+      load_ar_association(object, :project_staff_assignment_configs).exists?
```

Comment:
We don't really need the data loader here, since this isn't called in batch. Call the class method and add a comment that indicates it shouldn't be used in batch